jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBC.java

```diff
@@ -38,27 +44,49 @@
     
     // Timeout (in seconds) used to check if connection is still valid
     private static final int CHECK_CONNECTION_TIMEOUT = 5;
-    
+
+    // Timeout (in milliseconds) specifying how long ds.getConnection() will wait for a connection before timing out
+    private static final int CONNECTION_POOL_TIMEOUT = 5000;
+
+    // Used if asynchronous publish/query handling has been specified
+    private HikariConfig config = new HikariConfig();
```

Comment:
We have lots of `config`s.  Might be good to give this a more descriptive name.

Also, need we allocate it if we aren't going to use it?